Framework/DataObjects/src/EventWorkspace.cpp

```diff
@@ -186,6 +186,22 @@ const EventList &EventWorkspace::getSpectrum(const size_t index) const {
   return *data[index];
 }
 
+/**
+ * Returns a pointer to the EventList for a given spectrum in a timely manner.
+ *
+ * Very minimal checking is performed by this function and it should only be
+ * used in tight loops where getSpectrum is too costly.
+ *
+ * See the implementation of the non-const getSpectrum to see what checking is
+ * missing.
+ *
+ * @param index Workspace index
+ * @return Pointer to EventList
+ */
+EventList *EventWorkspace::getSpectrumQuickly(const size_t index) {
```

Comment:
I think less developers would mistakenly use this method if it were called `getSpectrumMinimalChecks`. The current name suggests everybody should use it everywhere which the documentation clearly states isn't the case.

Reply:
As much as developers that don't read documentation deserve exactly what they get, I've renamed it to `getSpectrumUnsafe` (not all missing things are checks).